lib/rpc/RpcMethods.js

```diff
@@ -54,10 +55,18 @@ class RpcMethods {
    */
   async placeOrder(params) {
     const { order } = params;
-    // temporary simple invoices until swaps are operational
-    const invoice = await this.lndClient.addInvoice(order.price * order.quantity);
-    order.invoice = invoice.payment_request;
-    await this.orderbook.addOrder(order);
+    assert(typeof order.price === 'number', 'price name must be a number');
+    assert(order.price > 0, 'price must be greater than 0');
+    assert(typeof order.quantity === 'number', 'quantity must be a number');
+    assert(order.quantity > 0, 'quantity must be not equal 0');
```

Comment:
@moshababo We accept negative quantity for sell orders, right?

Reply:
Yes, good catch, fix commited